torchvision/models/googlenet.py

```diff
@@ -84,34 +84,55 @@ def forward(self, x):
             x_ch2 = torch.unsqueeze(x[:, 2], 1) * (0.225 / 0.5) + (0.406 - 0.5) / 0.5
             x = torch.cat((x_ch0, x_ch1, x_ch2), 1)
 
+        # N X 3 X 224 X 224
```

Comment:
Using a lowercase `x` everywhere would make the comments clearer and more consistent with inceptionv3.

Reply:
Thanks for pointing out that!